components/Leaderboard/PodiumCard.vue

```diff
@@ -0,0 +1,215 @@
+<script setup>
+  import RichProfileLink from '~/components/Rich/RichProfileLink.vue';
```

Comment:
Components are auto-imported. Simply use `<rich-profile-link />` in the template.